manifests/backend.pp

```diff
@@ -118,11 +128,17 @@
   $api_protocol = $sensu::api_protocol
   $password = $sensu::password
 
-  if $use_ssl and ! $ssl_cert_source {
-    fail('sensu::backend: ssl_cert_source must be defined when sensu::use_ssl is true')
+  if $use_ssl and $ssl_cert_source and $ssl_cert_content {
+    fail('sensu::backend: Do not define both ssl_cert_source and ssl_cert_content_content')
   }
-  if $use_ssl and ! $ssl_key_source {
-    fail('sensu::backend: ssl_key_source must be defined when sensu::use_ssl is true')
+  if $use_ssl and ! ($ssl_cert_source and $ssl_cert_content) {
+    $ssl_cert_source = $facts['puppet_hostcert']
```

Comment:
Similar to my other commands, you can't redfine the same variable twice in Puppet. If $ssl_cert_content is undefined then use pick, else make the $_ssl_cert_source set to undefined.